src/app/home/home.component.html

```diff
@@ -1,14 +1,28 @@
-<div id="container">
-
-    <label> User Name</label>
-    <input type="text"/>
-    <label> Password</label>
-    <input type="password"/>
-
-    <h3><a routerLink="/dashboard">Login</a></h3>
-    <h3><a routerLink="">Register</a></h3>
-
-</div>
-
-
+<body>
+    <div class="container">
+        <!-- Left Column -->
+        <div class="left-column">
+            <img src="assets\images\bgrd.svg" alt="hello" />
+            <h3><b>RDCRMS</b></h3>
+            <p>Revenue department certificate registration<br>management system.We provide the certificates<br> and
+                provide the real-time monitoring.. </p>
+        </div>
 
+        <!-- Right Column -->
+        <div class="right-column">
+            <form class="f1">
+                <div class="cn">
+                    <label for="username"><b>Username or email</b></label><br>
+                    <input type="text" id="username" name="username" placeholder="johnsmith007" />
+                    <label for="password"><b>Password</b></label><br>
+                    <input type="password" id="password" name="password" placeholder="*********" />
+                    <a href="#">Forgot password?</a>
+                    <button type="submit">Sign in</button>
+                    <p class="or">------- or -------</p>
+                    <p class="sign">Sign in with Google</p>
+                </div>
+            </form>
+            <p class="footer">Are you new? <a href="#">Create an account</a></p>
```

Comment:
<a routerLink="/register">Create New Account</a>

Reply:
I can added router link for create new account once click on create new account it will navigate register page
![Screenshot (146)](https://github.com/user-attachments/assets/f70e43fd-d571-446d-9cec-61298a70c634)